specification/communication/data-plane/Chat/preview/2025-11-15-preview/communicationserviceschat.json

```diff
@@ -0,0 +1,2210 @@
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "chatImageFile",
+            "description": "Image binary data, allowed image formats: jpeg, png, gif, heic, webp",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ChatImageFile"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Uploaded successfully, the `Location` header contains the URL for the newly uploaded image.",
+            "schema": {
+              "$ref": "#/definitions/UploadChatImageResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Upload Image": {
+            "$ref": "./examples/Images_UploadChatImage.json"
+          }
+        }
+      }
+    },
+    "/chat/threads/{chatThreadId}/images/{imageId}/view/{imageViewType}": {
+      "get": {
+        "tags": [
+          "Images"
+        ],
+        "summary": "Get an image by view type.",
+        "operationId": "ChatThread_GetChatImage",
+        "produces": [
+          "application/json",
+          "image/gif",
+          "image/jpeg",
+          "image/png",
+          "image/bmp",
+          "image/tiff"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "chatThreadId",
+            "description": "The thread id to which the message was sent.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "imageId",
+            "description": "The image id.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "imageViewType",
+            "description": "The view type of image.",
+            "required": true,
+            "enum": [
+              "original",
+              "small"
+            ],
+            "type": "string",
+            "x-ms-enum": {
+              "name": "ImageViewType",
+              "modelAsString": true
+            }
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Image binary data",
+            "schema": {
+              "$ref": "#/definitions/ChatImageFile"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
```

Comment:
In existing APIs, the error response is `CommunicationErrorResponse`.

Any reason we switch here? (not against this, just trying to know why we switch now)